Kitodo/src/main/java/org/kitodo/production/forms/copyprocess/ProzesskopieForm.java

```diff
@@ -14,17 +14,23 @@
 import de.unigoettingen.sub.search.opac.ConfigOpac;
```

Comment:
I thought we are going to replace this file? but instead there is even a lot of new code here :(

Reply:
This class should be deleted. But I will delete all unused classes in a seperate pull request.